stories/modules-tagmanager-settings.stories.js

```diff
@@ -133,6 +133,15 @@ storiesOf( 'Tag Manager Module/Settings', module )
 
 		return <Settings isOpen={ true } isEditing={ true } registry={ registry } />;
 	} )
+	.add( 'Edit, with "Set up a new container"', ( registry ) => {
+		const webContainerVersion = fixtures.liveContainerVersions.web.gaWithVariable;
+		const accountID = webContainerVersion.accountId; // eslint-disable-line sitekit/camelcase-acronyms
+
+		registry.dispatch( STORE_NAME ).receiveGetAccounts( fixtures.accounts );
+		registry.dispatch( STORE_NAME ).receiveGetContainers( [], { accountID } );
+
+		return <Settings isOpen={ true } isEditing={ true } registry={ registry } />;
```

Comment:
Hah, just noticed that here (and elsewhere in this file) we're using `isOpen={ true }` rather than just `isOpen`. Hardly a big deal, but I'm surprised ESLint didn't warn us—I thought a lot of ESLint rules prefer boolean props to omit the `={ true }` part. 🤷🏻‍♂️

Reply:
I have updated these stories to use just `isOpen` and `isEditing`.